### PR TITLE
chore: Release 0.2.20

### PR DESCRIPTION
**Pull request by robinlinden**
This change is [Reviewable](https://reviewable.io/reviews/TokTok/c-toxcore/2792)

### CODE REVIEW

**Review by iphydf — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+- [#2777](https://github.com/TokTok/c-toxcore/pull/2777) chore: Upgrade to FreeBSD 14.1 in cirrus build.
+- [#2772](https://github.com/TokTok/c-toxcore/pull/2772) fix: friend_connections leak on allocation failure.
+- [#2771](https://github.com/TokTok/c-toxcore/pull/2771) fix: events leak that can occur if allocation fails
+- [#2769](https://github.com/TokTok/c-toxcore/pull/2769) chore(ci): new minimum for all android versions is 21
+- [#2768](https://github.com/TokTok/c-toxcore/pull/2768) fix: toxav rtp temp buffer allocation size was too large
+- [#2762](https://github.com/TokTok/c-toxcore/pull/2762) chore(cmake): set options changes as cache and with force
+- [#2761](https://github.com/TokTok/c-toxcore/pull/2761) chore: Fix CI
+- [#2757](https://github.com/TokTok/c-toxcore/pull/2757) fix: Use Opus in the CBR mode
+- [#2755](https://github.com/TokTok/c-toxcore/pull/2755) chore: Fix Circle CI build failing
+- [#2754](https://github.com/TokTok/c-toxcore/pull/2754) docs(toxav):  fix docs of toxav.h
```
Comment (nurupo) [inline CHANGELOG.md:25]: Where did this extra space came from? I don't see it in the PR title.

**Review by nurupo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (robinlinden) [reply CHANGELOG.md:25]: They're autogenerated, and I just regenerated them to check that I didn't mess up when moving files around. Maybe an invisible character in the title or something?

Comment (robinlinden) [reply CHANGELOG.md:25]: Oh, it contains 2 spaces. GitHub just hides one of them unless you edit the title.

Comment (robinlinden) [reply CHANGELOG.md]: Fixed it and regenerated the changelog. :P Please take another look.

Comment (nurupo) [reply CHANGELOG.md]: Okay, that explains it then.

>I don't see it in the PR title.

I also checked the email subject and it also had one space in it. Though I have now checked the actual email headers and apparently the subject is split in two lines, which doesn't happen with other PRs:

```
Subject: Re: [TokTok/c-toxcore] docs(toxav):
  fix docs of toxav.h (PR #2754)
```

Looks like @Green-Sky sneaked in some extra spaces / new-lines in there, heh.

**Review by nurupo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Green-Sky) [reply CHANGELOG.md]: oops, i changed the title to retrigger the merge bot. ... guess the extra space did exist after all

**Pull request merged**